script/update

```diff
@@ -236,12 +264,44 @@ def run_gn(target_arch, defines):
   else:
     target_cpu = target_arch
 
+  if target_cpu == 'mips64el':
+    subprocess.call(['chmod', '+x', os.path.join(SOURCE_ROOT, 'script', 'mips64el', 'ninja-mips64el')], env=env)
+    os.system('sh script/mips64el/ninja-mips64el')
+    env['PATH'] = os.pathsep.join([os.path.join(VENDOR_DIR, 'depot_tools'),
+                                   env['PATH']])
+    bootstrap = os.path.join(SRC_DIR, 'tools', 'gn', 'bootstrap', 'bootstrap.py')
+    subprocess.call([bootstrap, '-s'], cwd=SRC_DIR, env=env)
+    gn = os.path.join(SRC_DIR, 'out', 'Release', 'gn')
+
   for component in COMPONENTS:
     args = 'import("//chromiumcontent/args/{0}.gn") target_cpu="{1}"'.format(component, target_cpu)
     if sys.platform in ['win32', 'cygwin']:
       args += ' use_experimental_allocator_shim=false'
     if sys.platform == 'linux2':
       args += ' use_gtk3=false'
+      if target_arch == 'mips64el':
+        args += ' google_api_key="AIzaSyDxKL42zsPjbke5O8_rPVpVrLrJ8aeE9rQ"'
+        args += ' google_default_client_id="595013732528-llk8trb03f0ldpqq6nprjp1s79596646.apps.googleusercontent.com"'
+        args += ' google_default_client_secret="5ntt6GbbkjnTVXx-MSxbmx5e"'
```

Comment:
The Google API keys are not needed in libchromiumcontent.

Reply:
ok